kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -301,6 +359,9 @@ class NotificationManager internal constructor(
      * Create a media player notification that automatically updates.
      *
      * **NOTE:** You should only call this once. Subsequent calls will result in an error.
+     *
+     * Updated: Only update [internalNotificationManager] properties when call [createNotification] again.
+     * If you want to update the ResourceId of the buttons, call [hideNotification] before calling [createNotification] function.
```

Comment:
can we check somehow if the button icons changed and invalidate the internalNotificationManager in this case? Otherwise it is a 'breaking' change that you have to hide the notification in order to change the icons..